modules/release-notes/con-relnotes-fixed-issues.adoc

```diff
@@ -1,5 +1,37 @@
 [id='con-relnotes-fixed-issues_{context}']
-= Fixed issues
+= Fixed issues in {product} {product-version} and 1.2.2
+
+== Fixed issues in {product} 1.2.2
+
+This section lists fixed issues with {product} 1.2.2:
+
+Dynamic plugins fail to load if the file name in ConfigMap is invalid::
++
+--
+Previously, an invalid dynamic plugin file name in ConfigMap prevented the creation of a Backstage CR. With this release, the Backstage CR can be created even with an invalid plugin file name, and a corresponding log message will be displayed if the {product-short} build fails.
+--
+
+Plugin ID is missing in the permission table in RBAC Administration UI::
++
+--
+Previously, the plugin ID was missing from the permission table in the *Administration* -> *RBAC*. With this release, updates to both the RBAC backend and frontend plugins now ensure the plugin ID is displayed in the table.
```

Comment:
I'm assuming this refers to more than one plugin's ID, not just the single entry for the RBAC plugin? Unclear from the way this is written... 

```suggestion
Previously, the plugin ID was missing from the permission table in UI accessed from *Administration* -> *RBAC*. With this release, updates to both the RBAC backend and frontend plugins now ensure that plugin IDs are displayed in the table.
```

Reply:
Done